content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx

```diff
@@ -396,6 +396,8 @@ model CategoriesOnPosts {
   category   Category @relation(fields: [categoryId], references: [id])
   categoryId Int
   assignedAt DateTime @default(now())
+  assignedBy User     @relation(fields: [userId], references: [id])
+  userId     Int
```

Comment:
```suggestion
  createdAt     DateTime @default(now())
  visible     Boolean     
  userId     Int
```

Suggesting this change to avoid references to a `User` model that doesn't exist - see @janpio's comments above